carma-messenger-core/cpp_message/include/cpp_message.h

```diff
@@ -81,6 +83,12 @@ class Message
      * The encoded message is published as outbound binary message. Failure to encode results in a ROS Warning.
      */
     void outbound_mobility_response_message_callback(const cav_msgs::MobilityResponse& msg);
+    /**
+     * @brief function callback when there is an incoming mobility path message. .
```

Comment:
Should this say "outgoing" instead of incoming to match your pub/sub comments?

Reply:
Updated